### PR TITLE
Reverting /tg/'s bulky guns, or at least the ones I know of.

### PR DESCRIPTION
**Pull request by Pugie**
## About The Pull Request

Energy guns, bulldogs, wormhole guns, reverse engineered ebows and more can fit in backpacks again. Reverts several changes to item size from https://github.com/tgstation/tgstation/pull/54041 and https://github.com/tgstation/tgstation/pull/56132
Reverts unnecessary flavour texts from the latter too, they're a pet peeve that annoys me, the game already tells you the items are normal sized, why do you need to specify in the description too.

## Why It's Good For The Game

People carrying tons of guns is not an issue, carrying tons of guns in your backpack means you have nothing but guns and you just die. What bulky guns does do is prevent people from concealing weapons. This game is not about telegraphing every single attack you have to make things fair for other players, people will use dirty tactics including concealed weapons to get the drop on you and kill you. You will die to people with autorifles in their backpacks. Deal with it.

For the sake of "consistency" these PRs makes the life of antags much more difficult, why the hell is a bulldog bulky? I get mad just thinking about bulky guns. Oh my god.

## Changelog
:cl:
balance: most guns that weren't bulky before 2021 are debulked./:cl:

### CODE REVIEW

**Comment by Burgermancoding** [comment]
the debulkening!

**Comment by Pugie** [comment]
I cry thinking about the guy who said that bulldog should be bulky because bullpup =/= compact

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "A basic energy-based laser gun that fires concentrated beams of light which pass through glass and thin metal."
 	icon_state = "laser"
 	inhand_icon_state = "laser"
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = WEIGHT_CLASS_NORMAL
```
Comment (stanalbatross) [inline code/modules/projectiles/guns/energy/laser.dm:6]: will inherit from gun/energy, will it not?

Comment (Pugie) [reply code/modules/projectiles/guns/energy/laser.dm:6]: Yes, why was this ever there in the first place.

**Pull request merged**